src/web/server/locales/cy/common.json

```diff
@@ -154,9 +154,14 @@
     "title": "Sed do eiusmod tempor incididunt ut labore et dolore",
     "heading": "Sed do eiusmod tempor incididunt ut labore et dolore"
   },
+  "guidance": {
+    "title": "Ex monetaria mooeo, vitus et grub (Healthy Start)",
+    "navigationHeading": "Contentus"
+  },
   "yes": "Diam",
   "no": "Sed",
   "back": "Kcab",
   "text": "Occaecat",
-  "email": "Lobortis"
```

Comment:
Why was email removed? Guessing this wasn't intentional?

Reply:
Errrr .. no, that was a mistake. Have updated.